### ISSUE TITLE
Several links broken in README

### ISSUE THREAD

**Issue by tmadlener** [key4hep/key4hep-doc#77]
The current `build` workflow that is run for PRs is broken, because the link checker is not able to resolve links in the LICENSE file (it gets a `403` presumably in effort to avoid scraping). Additionally one for intels TBB seems to no longer work, see e.g.

```
(         LICENSE: line    3) broken    https://opensource.org/ - 403 Client Error: Forbidden for url: https://opensource.org/
(         LICENSE: line    3) broken    https://opensource.org/license/apache-2-0/ - 403 Client Error: Forbidden for url: https://opensource.org/license/apache-2-0/
(how-tos/key4hep-tutorials/edm4hep_analysis/edm4hep_api_intro: line   12) ok        https://key4hep.web.cern.ch/podio
(    CONTRIBUTING: line   16) ok        https://reflectoring.io/github-fork-and-pull/
```

https://github.com/key4hep/key4hep-doc/actions/runs/12049862155/job/33597515436

**Comment by m-fila** [comment]
Hmm I can't reproduce this locally
```txt
(talks-and-presentations/README: line   22) ok        https://indico.cern.ch/event/934666/contributions/4154228/
(  how-tos/README: line    3) ok        https://key4hep.github.io/key4hep-doc/tutorials/README.html
(         LICENSE: line    3) ok        https://opensource.org
(         LICENSE: line    3) ok        https://opensource.org/license/apache-2-0/
(developing-key4hep-software/SpackForDevelopers: line   60) ok        https://spack-tutorial.readthedocs.io/en/latest/tutorial_developer_workflows.html
```

**Comment by tmadlener** [comment]
Fixed by #79
